backend/src/main/java/com/example/backend/use_case/display_stops/StopsService.java

```diff
@@ -1,12 +1,9 @@
 package com.example.backend.use_case.display_stops;
 
-import com.example.backend.data_access.stop.StopDataAccessInterface;
 import com.example.backend.entity.Location;
-import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.stereotype.Service;
 
 import java.util.HashMap;
 
 public interface StopsService {
-    public HashMap<String, Location> execute();
+    HashMap<String, Location> execute();
```

Comment:
what's the benefit of removing the access modifier?

Reply:
It's an interface so you don't need it.